This exploded with a similar error (from https://treeherder.mozilla.org/#/jobs?repo=try&revision=fe980dd57b22a2814f99b85540e5341c5e40caa4&group_state=expanded&selectedJob=279625874):

```
[task 2019-12-04T22:25:50.051Z] Traceback (most recent call last):
[task 2019-12-04T22:25:50.051Z]   File "/builds/taskcluster/script.py", line 250, in <module>
[task 2019-12-04T22:25:50.051Z]     sys.exit(main())
[task 2019-12-04T22:25:50.051Z]   File "/builds/taskcluster/script.py", line 220, in main
[task 2019-12-04T22:25:50.051Z]     bytes_written += sys.stdout.write(decoded_line)
[task 2019-12-04T22:25:50.051Z] UnicodeEncodeError: 'ascii' codec can't encode character '\x80' in position 45: ordinal not in range(128)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -62,6 +62,8 @@ RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
 # Set variables normally configured at login, by the shells parent process, these
 # are taken from GNU su manual
+# - PYTHONIOENCODING is set so that unicode characters can be output
+#   - see https://bugzilla.mozilla.org/show_bug.cgi?id=1600833
 
 ENV    HOME=/builds/worker \
        SHELL=/bin/bash \

--- a/Dockerfile
+++ b/Dockerfile
@@ -68,6 +68,7 @@ ENV    HOME=/builds/worker \
        LANGUAGE=en_US.UTF-8 \
        LANG=en_US.UTF-8 \
        LC_ALL=en_US.UTF-8 \
+       PYTHONIOENCODING=utf-8 \
        PATH=$PATH:/builds/worker/bin
 
 # download things

--- a/scripts/entrypoint.py
+++ b/scripts/entrypoint.py
@@ -32,6 +32,7 @@ def dump_scriptvars():
         "HOST_IP",
         "HOSTNAME",
         "PATH",
+        "PYTHONIOENCODING",
         "TC_WORKER_GROUP",
         "TC_WORKER_TYPE",
         "TESTDROID_BUILD_ID",

